docs/Overview.md

```diff
@@ -0,0 +1,157 @@
+      category: 'rooms',
+      source: 'room views',
+      actions: ['get', 'add', 'remove']
+    });
+
+    //the dispatchGet input is the method that will actually dispatch our rooms/get action. Thus, here we are submitting an ajax request and then dispatching the results of that request for our rooms/get action whenever roomActions.get method is invoked
+    roomActions.before('get', function (dispatchGet) {
+      ajaxRequest('/rooms', 'GET').then(function (results) {
+        dispatchGet(results);
+      });
+    });
+
+    module.exports = roomActions;
+```
+
+### TuxedoJS makes it easy to create stores and hook them up with your actions
+TuxedoJS abstracts away all of the boilerplate associated with creating stores and makes it easy to connect stores with actions and views.
+
+```
+    var Stores = require('tux/Stores');
+    var roomActions = require('../actions/roomActions');
+
+    var roomStore = Stores.createStore({
+      _rooms = [],
+
+      getAll: function () {
+        return this._rooms;
+      },
+
+      onGetDispatch: function (roomsData) {
+        this._rooms = roomsData;
+        this.emitChange();
+      }
+    });
+
+    roomActions.register(roomStore, {
+      //registered callbacks are invoked with the context of the passed in store by default so we don't need to do any binding here
+      get: roomStore.onGetDispatch
+    });
+
+    module.exports = roomStore;
+```
+
+### TuxedoJS provides powerful augmented `React` syntax which makes it extremely easy to connect your state managing views with your stores
+TuxedoJS provides opinionated `React` classes with built in convenience methods for performing distinct operations in your app. The augmented class in the example below possesses tools for easily and semantically connecting with Tux Stores.
+
+Note here that we are storing the result of `require('tux/React')` in a variable called `React` here.  This is to take advantage of React's beautiful JSX syntax, which automatically converts JSX to `React.METHOD NAME`.
+
+```
+    var React = require('tux/React');
+    var roomStore = require('../stores/roomStore');
+    var roomActions = require('../actions/roomActions');
+    var Rooms = require('./Rooms'); //for this example let's imagine we have created a Rooms TuxReact class
+
+    var RoomViewOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          rooms: roomStore.getAll()
+        };
+      },
+
+      connectOwnerToStore: function () {
+        return {
+          store: roomStore,
+          listener: function () {
+            this.setState({
+              rooms: roomStore.getAll()
+            });
+          }.bind(this)
+        };
+      },
+
+      componentDidMount: function () {
+        roomActions.get();
+      },
+
+      render: function () {
+        return (
+          <Rooms rooms={this.state.rooms} />
+        );
+      }
+    });
+
+    module.exports = RoomViewOwner;
+```
```

Comment:
Do you want to go into detail about what the `OwnerClass` provides as well as the `MutableClass`?